packages/mds-metrics/utils.ts

```diff
@@ -1,16 +1,15 @@
 import { now, yesterday } from '@mds-core/mds-utils'
+
 import { GetTimeBinsParams } from './types'
 
-export function getTimeBins({
+export async function getTimeBins({
```

Comment:
Weirdly enough, I discovered not marking this function as `async` and `awaiting` it causes a whole bundle of issues... I'm guessing some weirdness with the array constructor method.